### DISCUSSION TITLE
Welcome to ish-AOK Discussions!

### DISCUSSION THREAD

**Discussion by emkey1** [emkey1/ish-AOK#156] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stone-90** [comment]
我使用您的ish-AOk的alpine linux发行版，安装完后用vim打开文件时提示E1286 could not set timeout：invalid argument，用vi无此问题，是vimrc配置的问题吗？
